Bottom: e3dcb43c603f81b23578234ff53792c7a723d65a
Top:    6105ad33c597dd6f1bbc0c24ab46290debbcaf8f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:26 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6e8c5d3..9f1d695 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2698,7 +2698,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
